test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
Not sure what to say.  This is a small subset of ShEx I have been writing for demographics CMUMPS to FHIR translation using patient-7 obfuscated test data.  Is that the comment you want?